src/agent/tests/CMakeLists.txt

```diff
@@ -1,11 +1,10 @@
-# Wazuh Agent tests
-
 find_package(GTest CONFIG REQUIRED)
 
 set(TEST_SOURCES
```

Comment:
I think `TEST_SOURCES` is not used so it can be removed.

Reply:
Same in `src/agent/agent_info/tests/CMakeLists.txt`.

Reply:
done 05bf17df02fec16afd3a4954836dd16a48723f70